warehouse/admin/templates/admin/users/list.html

```diff
@@ -56,11 +55,18 @@
         </td>
         <td>{{ user.name }}</td>
         <td>{{ user.email }}</td>
-        <td>{% if user.is_superuser %}<i class="fa fa-check text-green"></i>{% endif %}</td>
-        <td>{% if user.is_moderator %}<i class="fa fa-check text-green"></i>{% endif %}</td>
-        <td>{% if user.is_psf_staff %}<i class="fa fa-check text-green"></i>{% endif %}</td>
+        <td>
+          {% set pypi_roles = [] %}
+          {% if user.is_superuser %}{% set _ = pypi_roles.append("Admin") %}{% endif %}
+          {% if user.is_moderator %}{% set _ = pypi_roles.append("Moderator") %}{% endif %}
+          {% if user.is_psf_staff %}{% set _ = pypi_roles.append("PSF Staff") %}{% endif %}
```

Comment:
For reviewers - I opted to use `set _ = ` syntax instead of importing another jinja2 extension to use `do` instead. here's what that would look like:

```jinja2
{% if user.is_psf_staff %}{% do pypi_roles.append("PSF Staff") %}{% endif %}
```

If we have more use cases like `set _ =`, we could entertain adding the extension and using that syntax.